Source/AwsCommonRuntimeKit/io/ClientBootstrap.swift

```diff
@@ -31,7 +31,7 @@ public final class ClientBootstrap {
             user_data: callbackDataPointer
         )
         guard let rawValue = aws_client_bootstrap_new(allocator.rawValue, &options) else {
-            throw CRTError(errorCode: aws_last_error())
+            throw AWSCommonRuntimeError.CRTError(CRTError.init(fromErrorCode: aws_last_error()))
```

Comment:
It would be nice to create a convenience `init` for CRTError than repeating `CRTError.init(fromErrorCode: aws_last_error())`